dashboard/src/components/Header/ContextSelector.scss

```diff
@@ -0,0 +1,53 @@
+.angle {
+  margin-left: -0.8rem;
+  height: 100%;
+  &-closed {
+    transform: rotate(180deg);
+    margin-left: -1rem;
+  }
+  &-opened {
+    position: relative;
+    left: 10px;
+  }
```

Comment:
Did you try to use the `rotate` property of the icon component?

Reply:
mmh, I tried but there was no `rotate` property, turns out it's called `direction`

Reply:
lol 😂